translations/en.json

```diff
@@ -383,6 +383,28 @@
       }
     },
     "recovery": {
+      "progress": {
+        "change_email": "Try a different email",
+        "@change_email": {
+          "context": "pages/recovery/progress.html:38:33-72",
+          "description": "Button to change the email address for the password recovery link"
+        },
+        "description": "We send an email with a link to reset your password if there's an account using <span>%(email)s</span>.",
```

Comment:
sen**t**

Reply:
Done in c401045742f8821567a4b63bd2746a547185103d